addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -387,17 +387,19 @@ def _import_partner(self, company_id, name, phone, email, vat, country_code=Fals
 
     def _import_partner_bank(self, invoice, bank_details):
         """ Retrieve the bank account, if no matching bank account is found, create it """
-        bank_details = map(sanitize_account_number, bank_details)
+        # remove the journal from the context, as it may be set to import statements
+        Bank = self.env['res.partner.bank'].with_context(default_journal_id=False)
```

Comment:
No, I have reported it to the accounting channel a few months ago and it seems that has not been fixed.
In `_import_partner_bank` I was getting an error because `bank_details` was an iterable (once consume it is empty) and I think that I iterated multiple times during during optimization/query generation.

Reply:
I've opened at PR for this module. I'll remove the changes from here once it is merged up to master.
https://github.com/odoo/odoo/pull/187415